testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -154,7 +154,8 @@ def compute_kiwi_profile_version(host)
     next if node.nil?
     domain, _code = node.run("grep '^search' /etc/resolv.conf | sed 's/^search//'")
     conf = "DOMAIN='#{domain.strip}'\\nDEVICE='eth1'\\nSTARTMODE='auto'\\nBOOTPROTO='dhcp'\\nDNS1='#{proxy}'"
-    node.run("echo -e \"#{conf}\" > #{file} && echo -e \"#{conf2}\" > #{file2} && systemctl restart NetworkManager")
+    service = node.os_family =~ /^rocky/ ? 'NetworkManager' : 'network'
+    node.run("echo -e \"#{conf}\" > #{file} && echo -e \"#{conf2}\" > #{file2} && systemctl restart #{service}")
```

Comment:
Can we have as retail terminal a Rocky system?
This step definition might have and do more than one responsibility?
Maybe we want to offer this into our test framework, but not for retail terminal setup?

Reply:
No Retail is all about SLES.

Sorry if the context is misleading. We set up the RH-like minion to work with the Retail Proxy, but it remains a normal minion, not a reformatted terminal.